src/plugins/plugin.legend.js

```diff
@@ -101,19 +105,43 @@ class Legend extends Element {
 	constructor(config) {
 		super();
 
-		const me = this;
-		extend(me, config);
+		extend(this, config);
 
 		// Contains hit boxes for each dataset (in dataset order)
-		me.legendHitBoxes = [];
+		this.legendHitBoxes = [];
 
 		/**
  		 * @private
  		 */
-		me._hoveredItem = null;
+		this._hoveredItem = null;
 
 		// Are we in doughnut mode which has a different data type
-		me.doughnutMode = false;
+		this.doughnutMode = false;
+
+		this.chart = config.chart;
```

Comment:
I wonder if disabling the ts compiler option `strictPropertyInitialization` would allow omitting these.
>  Enable strict checking of property initialization in classes.

Reply:
Hmmm. It sure sounds like it would. But I tried it and it didn't have any effect and the docs also say it's set to `false` by default